### PR TITLE
chore: add connection metadata and output type

### PR DESCRIPTION
**Pull request by q2w**
This PR makes below changes,

1. Adds connection metadata manually to metadata.yaml
2. Adds output type manually to metadata.yaml

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -68,6 +68,7 @@ docker_test_integration:
 .PHONY: docker_test_lint
 docker_test_lint:
 	docker run --rm -it \
+	  -e ENABLE_BPMETADATA \
```
Comment (bharathkkb) [inline Makefile:71]: Let's remove this as discussed so changes are not overwritten until tooling is ready

**Review by bharathkkb — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (q2w) [reply Makefile]: I assume you mean to remove this from docker_generate_docs command which is responsible for generating metadata. I removed it from there. This is used for validation only and we can keep this here!

Comment (bharathkkb) [reply Makefile]: Ah yes, got confused with the diff. You are right lets keep it.

**Pull request merged**